libraries/bot-azure/pom.xml

```diff
@@ -73,6 +73,13 @@
       <artifactId>bot-dialogs</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-storage-blob</artifactId>
+      <version>12.10.0</version>
+      <scope>compile</scope>
+    </dependency>
+
```

Comment:
As soon as `AzureQueueStorage` is merged, we need to rebase this branch or the base one with `main` as this is already added there.

Reply:
That was `azure-storage-queue` if I am not mistaken, this is a different library

Reply:
You are right, however, we will probably have conflicts here after rebasing